db/versions/0064.yml

```diff
@@ -0,0 +1,124 @@
+version: 64
+description: Add `queue_worker_seen` and `queue_task_queue_seen` functions
+methods:
+  create_task_queue: {deprecated: true}
+  update_task_queue: {deprecated: true}
+  task_queue_seen:
+    description: |-
+      Recognize that a task queue has been seen, creating it if necessary, updating
+      its properties if not null, and in any case bumping its last seen time time. 
+      The expiration time is not allowed to move backward.
```

Comment:
Is a test needed that setting an earlier expiration time leaves it at the later value?

Reply:
Good point --I'll add one before pushing.

Reply:
Ah, this already exists, as a smaller part of a (admittedly fairly long, non-unit-y) larger test:
```
      await taskQueueSeen({
        // try to set expires backward                                                                                            
        expires: expires1,
        description: null,
        stability: null,
      });  
```